appveyor.yml

```diff
@@ -1,29 +1,53 @@
 build: false
-platform: x86
+clone_depth: 1
 clone_folder: c:\projects\symfony-demo
 
 cache:
-  - '%LOCALAPPDATA%\Composer\files'
```

Comment:
keeping the composer cache would still be a good idea though (Symfony does not do it due to the magic it does regarding subtree packages built on the fly, but you can do it)